public/assets/extra-libs/materialize-stripped/js/global.js

```
@@ -0,0 +1,427 @@
+})(window);
+
+// AMD
+if (typeof define === 'function' && define.amd) {
+  define('M', [], function() {
+    return M;
+  });
+
+  // Common JS
+} else if (typeof exports !== 'undefined' && !exports.nodeType) {
+  if (typeof module !== 'undefined' && !module.nodeType && module.exports) {
+    exports = module.exports = M;
```

Biome (2.1.2)

[error] A global variable should not be reassigned.

Assigning to a global variable can override essential functionality.

(lint/suspicious/noGlobalAssign)